indexer/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "indexer-explorer-lake"
```

Comment:
Haven't dropped `-lake` here, probably makes sense to keep it to easily differentiate?

Reply:
I guess we can drop the `-lake`. The changelog should give us a clear understanding since the `0.11` version Indexer for Explorer is based on Lake Framework.

Reply:
Good point, I'll remove it